src/libraries/System.Diagnostics.PerformanceCounter/tests/ZeroDataException.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Diagnostics.Tests
```

Comment:
I don't think it's worth creating a new exception type for this. Especially because any exception type will have exactly the same  effect. 

Just `Assert.InRange(idcc.Values.Count, 1, int.MaxValue);` should do it. It will throw an XunitException.